content/02.introduction.md

```diff
@@ -45,4 +45,5 @@ This paper seeks to accelerate the label function creation process by measuring
 We hypothesized that sentences describing one relationship type might share linguistic features such as keywords or sentence structure with sentences describing other relationship types.
 If this hypothesis were to, one could drastically reduce the time needed to build a relation extractor system and swiftly populate large databases like Hetionet v1.
 We conducted a series of experiments to estimate how label function reuse enhances performance over distant supervision alone.
+As biomedical text comes in various forms (e.g. publications, clinician notes, image captions, etc.), we chose to subset this space to only include open-access publications.
```

Comment:
```suggestion
As biomedical data comes in various forms (e.g. publications, electronic health records, images, genomic sequences, etc.), we chose to subset this space to only include open-access biomedical publications available on pubmed.
```
It looks like they're thinking broader than even text - see if you think this keeps the meaning you were going for.